javascript/packages/eslint-config-nebenan/index.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     'react/jsx-one-expression-per-line': 'off',
     'react/jsx-props-no-multi-spaces': 'off',
     'react/no-this-in-sfc': 'off', // broken
+    'react/react-in-jsx-scope': 'off', // useless with new react transform
```

Comment:
If possible, we should show warnings for react imports we don't need

Reply:
it will be caught by unused vars